### PR TITLE
Fix Error: uninitialized constant OS::Mac::CLT [Linux]

### PR DESCRIPTION
**Pull request by sjackman**
Fixes https://github.com/Linuxbrew/brew/issues/874

### CODE REVIEW

**Comment by sjackman** [comment]
This PR can be squash merged.

**Comment by sjackman** [comment]
Merged!

**Comment by scpeters** [comment]
Thanks!

**Comment by sjackman** [comment]
Thanks for the bug report, Steven. Let me know if you run into anymore trouble with Homebrew/brew on Linux.

File: Formula/gcc@5.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def osmajor
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? do
     reason "The bottle needs the Xcode CLT to be installed."
-    satisfy { MacOS::CLT.installed? }
+    satisfy { !OS.mac || MacOS::CLT.installed? }
```
[the remark this answers is not in the document]
Comment (iMichka) [reply Formula/gcc@5.rb]: I already fixed this :)

[the remark this answers is not in the document]
Comment (scpeters) [reply Formula/gcc@5.rb]: sorry; I can read but I apparently don't always :)

thanks for fixing it

[the remark this answers is not in the document]
Comment (sjackman) [reply Formula/gcc@5.rb]: That was my bug. Thanks for finding it and fixing it, all.

**Pull request merged**